horace_core/utilities/range_add_border.m

```diff
@@ -19,6 +19,11 @@
 % -------
 %   range      Expanded range
 
+if nargin == 1 % add epsilon-sized border
```

Comment:
Would be clearer as "if tol not defined" structure, rather than counting args.

Reply:
its difficult question -- the suggestion is an alternative way of doing that, though the provided way is clearly faster. the suggested check will work in this case, but in general it may be confusing. e.g. if you have func(a,b,c), use if exist('b','var') and call function with func(a,c), exist('b','var') will return true. This is why I am trying avoiding var checks. 

Anyway, as I've said replying to another comment, this function should be refactored to be more consistent, better verified and belongs to Herbert. This task worth another ticket due to various side-effects possible